<documentation title="Hooks Inline DocBlock Standard">
    <standard>
        <![CDATA[
    All WordPress hook execution functions should be preceded by a DocBlock explaining the hook.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: using a DocBlock to describe a hook.">
            <![CDATA[
/**
 * Description of hook.
 *
 * @since 1.0.0
 */
do_action( 'hook' );
        ]]>
        </code>
        <code title="Invalid: not using a DocBlock comment">
            <![CDATA[
/*
 * This is not a docblock.
 */
do_action( 'hook' );
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: including a @since tag with a valid X.Y.Z version.">
            <![CDATA[
/**
 * Summary of hook.
 *
 * <em>@since 1.0.0</em>
 */
do_action( 'hook' );
        ]]>
        </code>
        <code title="Invalid: including a @since tag with a different undeclared version.">
            <![CDATA[
/**
 * Description of hook.
 *
 * <em>@since Initial release.</em>
 */
do_action( 'hook' );
        ]]>
        </code>
    </code_comparison>
</documentation>
